### ISSUE TITLE
Enhancment suggestion 

### ISSUE THREAD

**Issue by ghost** [MyRequiem/LFS#1]
In LFS\stage-1-lfs-stable-x86_64\stage-2-build_temp_system\stage-1 the queue file can be made into a bash script that runs them all in order automatically like so https://pastebin.com/raw/uFxFnnZN

Ive changed mine to work this way and it seems to work well. Just trying to share whats made my life easier :)

**Comment by MyRequiem** [comment]
Yes, I do that when I build the entire system on a separate machine. In the
Vim editor, such a script is created from a 'queue' file in a few minutes :)
I wanted to add such a script, but only when the whole system is created. There
is still a lot missing.

**Comment by ghost** [comment]
Cool. Do you have a chat program you prefer? I dont want to clutter up your issues with questions and such lol.

**Comment by MyRequiem** [comment]
WhatsApp ?

**Comment by ghost** [comment]
FYI I fixed this script for the latest 10.1 book https://github.com/BimboPolitics/LFS/blob/master/lfs/stage-2/stage-2.1/file.sh

**Comment by ghost** [comment]
This one too :) https://github.com/BimboPolitics/LFS/blob/master/lfs/stage-2/stage-2.1/binutils-pass-2.sh

**Comment by MyRequiem** [comment]
Oh, yes :) Thank you very much, I did not notice these changes. Fixed in commits 97320d9 and a438dae

**Comment by ghost** [comment]
MyRequiem, Telegram ? ))

**Comment by MyRequiem** [comment]
oh well, you're fast))

+79170915668

**Comment by MyRequiem** [comment]
@myrequiem 
my.requiem

**Comment by MyRequiem** [comment]
well or like this
https://t.me/myrequiem

**Comment by MyRequiem** [comment]
I have to offer to do this, but I do not see the point in this. Anyone who knows the editor will do it in 1 minute.
